content/en/docs/release-notes/release-preview.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Release Preview
+description: "Learn about some of the upcoming changes to Spinnaker and their impact."
+---
+
+## Overview
+
+Learn about some of the upcoming major changes to Spinnaker by the open source community and Armory. Included in this page are the implications of upcoming changes. Armory will incorporate this information into the release notes when appropriate, but this page is meant to be a preview and advance notice.
+
+This page is divided between a preview of Open Source Spinnaker and Armory's extensions of it.
+
+## Open Source
+
+### Spinnaker services in version X.Y.
+
+This section describes the notable upcoming changes to Spinnaker services:
+
+#### Gate
+
+**Change**: Lore te gatesum
+* **Impact**: Make sure you hit 88 MPH before Biff shows up.
+
+
+**Change**: Lore te gatesum
+* **Impact**: Make sure you hit 88 MPH before Biff shows up.
+
+**Change**: Lore te gatesum
+* **Impact**: Make sure you hit 88 MPH before Biff shows up.
+
+#### Fiat
+
+**Change**: Lore te fiatsum
+* **Impact**: Don't let Gizmo get wet.
+
+#### Igor
+
+**Change**: Lore te igor
+* **Impact**: Don't cross the streams.
+
+### Breaking Change for Kubernetes Run Job stage
+
+#### Change
+
+Spinnaker no longer automatically appends a unique suffix to the name of jobs created by the Kubernetes Run Job stage. Prior to this release, if you specified `metadata.name: my-job`, Spinnaker would update the name to `my-job-[random-string]` before deploying the job to Kubernetes. As of 1.22, the job’s name will be passed through to Kubernetes exactly as supplied.
+
+To continue having a random suffix added to the job name, set the `metadata.generateName` field instead of `metadata.name`, which causes the Kubernetes API to append a random suffix to the name.
+
+This change is particularly important for users who are using the preconfigured job stage for Kubernetes, or who are otherwise sharing job stages among different pipelines. In these cases, jobs are often running concurrently, and it is important that each job have a unique name. In order to retain the previous behavior, these users will need to manually update their Kubernetes job manifests to use the generateName field.
+
+#### Impact
+
+Users of Spinnaker >= 1.20.3 can opt in to this new behavior by setting `kubernetes.jobs.append-suffix: false` in their `clouddriver-local.yml`.
+
+As of Spinnaker 1.22, this new behavior is the default. Users can still opt out of the new behavior by setting `kubernetes.jobs.append-suffix: true` in their `clouddriver-local.yml`. This will cause Spinnaker to continue to append a suffix to the name of jobs as in prior releases.
```

Comment:
```suggestion
As of Spinnaker 1.22, this new no-suffix behavior is the default. Users can still opt out of the new behavior, and retain the addition of a random suffix, by setting `kubernetes.jobs.append-suffix: true` in their `clouddriver-local.yml`. This will cause Spinnaker to continue to append a suffix to the name of jobs as in prior releases.
```